recipes/py-gfm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "py-gfm" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "f107e43248eba6236b19ceda78531f40c7bdb85ba4a219f478c92920397f4f98" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed
+  
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - markdown
+  imports:
+    - gfm
+    - mdx_gfm
+    - mdx_partial_gfm
+
+
+about:
+  home: https://github.com/zopieux/py-gfm
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: An implementation of Github-Flavored Markdown written as an extension to the Python Markdown library.
+  description: |
+    'An implementation of Github-Flavored Markdown written as an extension to the Python Markdown library.'
+  dev_url: https://github.com/ESSS/qt_style_sheet_inspector
```

Comment:
Is this the correct `dev_url`? It seems like it should be:

`dev_url: https://github.com/Zopieux/py-gfm`

Reply:
I was using the recipe from another package and did notice =x